android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java

```diff
@@ -135,6 +135,7 @@ public boolean onTouchEvent(MotionEvent event) {
 		tv.setKeyListener(null);
 		tv.setFocusable(false);
 		tv.setSingleLine(false);
+		tv.setEllipsize(ellipsize);
```

Comment:
I think it will just be a "good" change:
![ell](https://cloud.githubusercontent.com/assets/4334997/16463240/376ce21e-3e35-11e6-86ef-3c1d45c8f567.png)

it will just affect the fields where you have a fixed width (no wordWrap) and the text would be out of the label anyway. That's mostly the point where you are searching on how to switch on ellipsize :)
But since Android can switch of the ellipsizing it might have a different default value. I would prefer `truncateAt.END` because it will make the label look the same on both platforms from the beginning. You decide ;)